modules/simulator.js

```diff
@@ -23,6 +23,19 @@ $(function()
     e.setAttribute("height", land.height);
   }).resize();
 
+  // Vision & Vector toggle
+  $(window).click(function()
+  {
+    if(Debug.VISIONHIDE == false)
+    {
```

Comment:
I have this task in my Stats branch https://github.com/CrypticDuo/Evolution/pull/6